akka-persistence-testkit/src/main/scala/akka/persistence/testkit/state/PersistenceTestKitDurableStateStoreProvider.scala

```diff
@@ -14,9 +14,10 @@ import akka.persistence.testkit.state.javadsl.{
 }
 
 class PersistenceTestKitDurableStateStoreProvider(system: ExtendedActorSystem) extends DurableStateStoreProvider {
-  private def _scaladslDurableStateStore[T]() = new PersistenceTestKitDurableStateStore[T](system)
-  override def scaladslDurableStateStore(): DurableStateStore[Any] = _scaladslDurableStateStore[Any]()
+  private val _scaladslDurableStateStore = new PersistenceTestKitDurableStateStore[Any](system)
```

Comment:
There is a similar problem in `PersistenceTestKitReadJournalProvider`, could you fix that at the same time?

Reply:
Just pushed - will check checkmarks in my AM :)